apps/api/src/components/Authentication/OTPService.ts.ts

```diff
@@ -0,0 +1,43 @@
+import authclientStytch from "../../lib/stytch.js";
+import { normalizePhoneNum } from "../../helpers/phoneHelper.js";
+import AppError from "@/lib/AppError.js";
```

Comment:
Be consistent with your imports.
```suggestion
import authclientStytch from "@/lib/stytch.js";
import { normalizePhoneNum } from "@/helpers/phoneHelper.js";
import AppError from "@/lib/AppError.js";
```

Reply:
My bad the path alias was causing issues